tests/integration_tests/datasets/commands_tests.py

```diff
@@ -587,6 +618,7 @@ def test_create_dataset_command(self):
             self.assertEqual([owner.username for owner in table.owners], ["admin"])
 
         db.session.delete(table)
+        db.session.commit()
```

Comment:
Is this intermediate commit really needed? I would assume not, or at most a flush. But do correct me if I'm wrong here!

Reply:
This test case will raise `sqlalchemy.exc.OperationalError` exception if the commit operation is not included:
```python
    def test_create_dataset_command(self):
        ...
        with examples_db.get_sqla_engine() as engine:
            engine.execute("DROP TABLE IF EXISTS test_create_dataset_command")
            engine.execute(
                "CREATE TABLE test_create_dataset_command AS SELECT 2 as col"
            )
        ...
        db.session.delete(table)
        with examples_db.get_sqla_engine() as engine:
>           engine.execute("DROP TABLE test_create_dataset_command")

    def do_execute(self, cursor, statement, parameters, context=None):
>       cursor.execute(statement, parameters)
E       sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked
E       [SQL: DROP TABLE test_create_dataset_command]
E       (Background on this error at: https://sqlalche.me/e/14/e3q8)
```

After reviewing the code, I realized that the test case requires committing the new table before dropping it. I'm not sure why this wasn't caught earlier, but I've double-checked the new test case I added, which includes all necessary commits. To ensure the unit test passes, I've made this change. If I've missed anything, please let me know and I'll be happy to make adjustments.